pyaedt/edb_core/stackup.py

```diff
@@ -572,16 +576,31 @@ def import_stackup(self, fpath):
         return True
 
     @pyaedt_function_handler
-    def export_stackup(self, fpath, file_format="csv"):
-        """Export stackup definition to csv file.
+    def export_stackup(self, fpath, file_format="csv", include_material_with_layer=False):
+        """Export stackup definition to csv or json file.
 
         Parameters
         ----------
         fpath : str
-            File path to csv file.
+            File path to csv or json file.
         file_format : str, optional
             The format of the file to be exported. The default is ``"csv"``. Options are ``"csv"``, ``"xlsx"``.
+            ``"json"``.
+        include_material_with_layer : bool, optional.
+            Only used when json file are exported. ``True`` will include material definition inside layer ones.
+            ``False`` will keep the material definition section in the json file. Default value is ``False``.
+
         """
+        if file_format.lower() == "csv" or "xlsx":
+            return self._export_layer_stackup_to_csv_xlsx(fpath, file_format)
+        elif file_format.lower() == "json":
+            self._export_layer_stackup_to_json(fpath, include_material_with_layer)
+        else:
+            self._logger.warning("layer stackup format not supported, skipping import.")
```

Comment:
```suggestion
            self._logger.warning("Layer stackup format is not supported. Skipping import.")
```
Edits are made because warning messages should be complete sentences that include punctuation.